GeoLib/AABB.h

```diff
@@ -85,6 +85,20 @@ class AABB
 		}
 	}
 
+	/**
+	 * Construction of object using input pointer array.
+	 * @param pnt_array  an array of point pointers
+	 * @param size       size of the array
+	 */
+	AABB(PNT_TYPE * const* pnt_array, unsigned size) :
```

Comment:
For instance this constructor is used in `NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping-impl.h` in `computeMappingMatrices`. I think line 72 (`GeoLib::AABB<MeshLib::Node> aabb(ele.getNodes(), ele.getNNodes());`) could be substituted by `GeoLib::AABB<MeshLib::Node> aabb(ele.getNodes().begin(), ele.getNodes().end());` that reuses the existing constructor.

Apart from this, for C arrays it is possible to use `std::begin()`and `std::end()` to obtain iterators that are necessary for the existing constructor.